src/vs/workbench/browser/layout.ts

```diff
@@ -470,7 +492,11 @@ export class WorkbenchLayout implements IVerticalSashLayoutProvider, IHorizontal
 		}
 
 		// Activity Bar Part
-		this.activitybar.getContainer().size(null, activityBarSize.height);
+		if (windowConfig && windowConfig.window.macOSUseInlineToolbar) {
```

Comment:
Instead of duplicating the same code, the `activityBarSize` should be used in both cases. I know it was not used before but I do not think it would cause any harm.